src/output.rs

```diff
@@ -353,24 +353,91 @@ impl fmt::Display for DebugBuffer {
     }
 }
 
-fn format_bytes(data: &[u8], f: &mut fmt::Formatter<'_>) -> fmt::Result {
+fn format_bytes(data: &[u8], f: &mut impl fmt::Write) -> fmt::Result {
     #![allow(clippy::assertions_on_constants)]
-    const MIN_OVERFLOW: usize = 8192;
-    const MAX_START: usize = 2048;
-    const MAX_END: usize = 2048;
-    const MAX_PRINTED: usize = MAX_START + MAX_END;
-    assert!(MAX_PRINTED < MIN_OVERFLOW);
-
-    if data.len() >= MIN_OVERFLOW {
-        write!(
-            f,
-            "<{} bytes total>{:?}...<{} bytes omitted>...{:?}",
-            data.len(),
-            data[..MAX_START].as_bstr(),
-            data.len() - MAX_PRINTED,
-            data[data.len() - MAX_END..].as_bstr(),
-        )
+
+    const LINES_MIN_OVERFLOW: usize = 40;
+    const LINES_MAX_START: usize = 10;
+    const LINES_MAX_END: usize = 10;
+    const LINES_MAX_PRINTED: usize = LINES_MAX_START + LINES_MAX_END;
```

Comment:
> Looks good, just two tweaks.
> 
>     1. I think we should tweak defaults a bit:
>        ```rust
>        const LINES_MIN_OVERFLOW: usize = 100;
>        const LINES_MAX_START: usize = 25;
>        const LINES_MAX_END: usize = 50;  // tail is more likely to contain error in CLI
>        ```

I feel like 100 lines is going to be too long but I agree about extending the size of `END`

Reply:
I went with 80, 20, and 40 (respectively). But I don't have strong feelings about these, and would be happy to adjust.